src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp

```diff
@@ -53,9 +53,11 @@ namespace picongpu
             //gaussian beam waist in the nearfield: w_y(y=0) == W0
             const float_64 w_y = W0 * sqrt( 1.0 + ( FOCUS_POS / y_R )*( FOCUS_POS / y_R ) );
```

Comment:
could you cast `W0` to `float_64` here?
**or better**
remove `W0 * ... ` here and `float_64( W0 ) / w_y` in both cases below